presentation/src/main/java/com/stop/AlarmWorker.kt

```diff
@@ -0,0 +1,55 @@
+package com.stop
+
+import android.content.Context
+import android.util.Log
+import androidx.hilt.work.HiltWorker
+import androidx.work.CoroutineWorker
+import androidx.work.Data
+import androidx.work.WorkerParameters
+import androidx.work.workDataOf
+import com.stop.domain.model.nearplace.Place
+import com.stop.domain.usecase.nearplace.GetNearPlacesUseCase
+import dagger.assisted.Assisted
+import dagger.assisted.AssistedInject
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.withContext
+
+// 일단은 NearPlaceUseCase 이용하여서 데이터 가져오기 구현
+
+@HiltWorker
+class AlarmWorker @AssistedInject constructor(
+    @Assisted context: Context,
+    @Assisted workerParameters: WorkerParameters,
+    private val getNearPlacesUseCase: GetNearPlacesUseCase
+) : CoroutineWorker(context, workerParameters) {
+
+    lateinit var resultList: List<Place>
```

Comment:
```suggestion
    lateinit var places: List<Place>
```
저희 컨벤션에서는 이런식으로 작성하도록 되어 있습니다.

Reply:
해당 리스트값이 필요가없고 단순 API 호출이 잘되었는지 확인할려는 코드인지라 나중에 삭제될예정이에요 !!